lib/stripe.rb

```diff
@@ -119,6 +119,49 @@ def self.set_app_info(name, partner_id: nil, url: nil, version: nil)
       version: version,
     }
   end
+
+  class Preview
+    def self._get_default_opts(opts)
+      { api_mode: :preview }.merge(opts)
+    end
+
+    def self.get(url, opts = {})
+      Stripe.raw_request(:get, url, {}, _get_default_opts(opts))
+    end
+
+    def self.post(url, params = {}, opts = {})
+      Stripe.raw_request(:post, url, params, _get_default_opts(opts))
+    end
+
+    def self.delete(url, opts = {})
+      Stripe.raw_request(:delete, url, {}, _get_default_opts(opts))
+    end
+  end
+
+  class RawRequest
+    include Stripe::APIOperations::Request
+
+    def initialize
+      @opts = {}
```

Comment:
Is this actually necessary?

Reply:
The implementation of Stripe::APIOperations::Request does https://github.com/stripe/stripe-ruby/blob/master/lib/stripe/api_operations/request.rb#L117